hooks/sentry/sentry.go

```diff
@@ -60,8 +60,9 @@ type SentryHook struct {
 	// consider the message correctly sent
 	Timeout time.Duration
 
-	client *raven.Client
-	levels []logrus.Level
+	client          *raven.Client
+	levels          []logrus.Level
+	stacktraceLevel *logrus.Level
```

Comment:
I wanted a way to allow people to skip enabling stacktrace without introducing another variable... will rethink the approach.